helm/aws-load-balancer-controller/values.yaml

```diff
@@ -26,6 +26,12 @@ serviceAccount:
   name:
   # Automount API credentials for a Service Account.
   automountServiceAccountToken: true
+  # List of image pull secrets to add to the Service Account.
+  # e.g.
+  # imagePullSecrets
+  # - name: docker
+  imagePullSecrets:
+  - name: docker
```

Comment:
default should be empty

Reply:
Shoot, I left this on for testing that the values piped through correctly. I've updated the values and comments as you've suggested.